docs/field-details.asciidoc

```diff
@@ -807,6 +807,76 @@ example: `docker`
 
 |=====
 
+[[ecs-dataset]]
+=== Dataset Fields
+
+The dataset fields are part of the new [indexing strategy](https://github.com/elastic/kibana/blob/master/docs/ingest_manager/index.asciidoc#indexing-strategy-1).
```

Comment:
This link is broken.

Reply:
Will update it but also need to find a more stable place. I wonder if I should link to a specific version instead.